governance/api.go

```diff
@@ -341,7 +350,11 @@ func (api *PublicGovernanceAPI) NodeAddress() common.Address {
 }
 
 func (api *PublicGovernanceAPI) isGovernanceModeBallot() bool {
-	gMode := api.governance.Params().GovernanceModeInt()
+	blockNumber := api.governance.BlockChain().CurrentBlock().NumberU64()
+	pset, err := api.governance.ParamsAt(blockNumber)
+	if err != nil {
+	}
```

Comment:
```suggestion
	if err != nil {
		return false
	}
```

Reply:
I forgot to return because it only returned bool. It seems reasonable to return false because the caller will handle error. fixed in 354eb20